packages/backend-common/src/reading/types.ts

```diff
@@ -97,3 +90,16 @@ export type ReadTreeResponse = {
    */
   etag: string;
 };
+
+export type ReadTreeResponseDirOptions = {
+  /** The directory to write files to. Defaults to the OS tmpdir or `backend.workingDirectory` if set in config */
+  targetDir?: string;
+};
+
+/**
+ * Represents a single file in a response.
+ */
+export type ResponseFile = {
```

Comment:
Not sure about the value of renaming this tbh, it couples them together and could make it more awkward to add something like `mode` in the future, which may be available in the archive but not search

Reply:
Sure! I can make it two separate types.